### PR TITLE
Retry and backoff when lost connection with containerd.

### PR DESCRIPTION
**Pull request by Random-Liu**
Retry to reconnect containerd event stream when containerd is down.

Signed-off-by: Lantao Liu <lantaol@google.com>

### CODE REVIEW

**Comment by Random-Liu** [comment]
Currently, if containerd is down when `cri-containerd` is running, `cri-containerd` will start printing error log in an infinity loop, which is quite spammy.

This PR also fixed that.

File: pkg/server/service.go [hunk 1]
```diff
@@ -129,5 +129,6 @@ func NewCRIContainerdService(conn *grpc.ClientConn, rootDir, networkPluginBinDir
 }
 
 func (c *criContainerdService) Start() error {
-	return c.startEventMonitor()
+	c.startEventMonitor()
+	return nil
```
Comment (mikebrow) [inline pkg/server/service.go]: nit.. don't need an error now...

Comment (Random-Liu) [inline pkg/server/service.go]: May need in the future. But I'm fine with removing it for now.

Comment (Random-Liu) [inline pkg/server/service.go]: Done

**Comment by Random-Liu** [comment]
Apply LGTM based on https://github.com/kubernetes-incubator/cri-containerd/pull/59#pullrequestreview-41367282.

**Pull request merged**